_submissions/round-12/0/2015-02-26-about-horacio-vargas.md

```diff
@@ -0,0 +1,12 @@
+---
+date: 2015-02-26
+round: Round 12
+title: About Horacio Vargas	
+author: 
+permalink:
```

Comment:
Please create a permanent link such as `/2015/02/about-horacio-vargas` (you can see other files in this directory for examples)